db/migrate/20151120153254_delete_bootable_interface.rb

```diff
@@ -0,0 +1,11 @@
+class DeleteBootableInterface < ActiveRecord::Migration
+  def up
+    Nic::Base.where(:type => 'Nic::Bootable').each do |nic|
```

Comment:
I think we shouldn't try to run validations. That's the point of using fake models, the reason is mixing versions of models and DB schema. Assume this migration is version 1, we'll add another column e.g. user_id in version 2 and we also add a validation for presence of user_id value. Now if we run model validation in version 1 we'd get errors on unknown user_id column since version 2 is not up yet.

So I think that the best we can do is using `update_all` to change the type.